### PR TITLE
add zh translation

### PR DESCRIPTION
**Pull request by alphacute**
add zh translation

### CODE REVIEW

File: i18n/zh/white-paper.md [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+
+| Term | Definition |
+|------|------------|
+| Leaf | The smallest denomination of the tea token. A leaf corresponds to one one-hundred-millionth ($10^{-8}$) of a tea. |
+| Slashing | The action of penalizing steepers or stakers in response to behavior contrary to the network rules. |
+| Staking | The action of locking tea tokens to secure the proof-of-stake network upon which the tea system is built. |
+| Steeping | The action of locking tea tokens to support your claim and receive rewards (or penalties) based on the consensus on the validity of your claim. |
+
+
+# References
```
Comment (mfts) [inline i18n/zh/white-paper.md:375]: I think you missed the translations for this section here

File: i18n/zh/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  tea incentivizes the maintenance of open-source by allowing network
+  participants to stake value against the packages they depend on and want to
+  secure. The tea protocol’s graph provides immutable package registration,
+  dependency requirements, package authenticity, and usage oracles to inform
+  the tea remuneration algorithm. Systematic inflation is distributed to all
+  packages based on that algorithm. If security or development issues are
+  found, developers can make claims supported by evidence against the package,
+  and slashing may occur. Members of the open-source community can review
+  packages for quality issues, and the protocol can respond to these reviews
+  by enacting proportional slashing events.
```
Comment (mfts) [inline i18n/zh/metadata.yml:21]: Please also translate the title and abstract

**Review by mfts — CHANGES_REQUESTED** [review]
Some parts are missing translation

**Comment by mfts** [comment]
Thank you @alphacute for the submission. We already have a Chinese translation from here: #78 and #79; perhaps you and @lovecity08 can work together on your translation.

**Comment by lovecity08** [comment]
I have been in contact with her and reviewed the Chinese translation

**Comment by mfts** [comment]
Thank you @alphacute for your contribution. We can only publish one translation per language and @lovecity08 submitted his first. I have since merged it into #92 and will be public soon. You are more than welcome contribute the Chinese translation once it's published or to any another language. 

We will make sure to mention you as an honorable contributor.

**Pull request closed without merging**